tests/test_json_validation.py

```diff
@@ -0,0 +1,230 @@
+from pathlib import Path
+from jsonschema import ValidationError
+
+import pytest
+from dpytools.validation.json.validation import validate_json_schema
+
+
+def test_validate_json_schema_data_path():
```

Comment:
you dont need to assert none. Just run the function, if. it raises the test will fail with that exception - thats fine.